.konflux/git-cloner/Dockerfile

```diff
@@ -12,22 +12,27 @@ RUN \
   microdnf --assumeyes --nodocs install git git-lfs && \
   microdnf clean all && \
   rm -rf /var/cache/yum && \
+  # The following setup is necessary so that this image can run as any user
+  mkdir -p /shared-home/.docker /shared-home/.ssh && chmod -R 0777 /shared-home && \
+  # This is the default user that will be used when strategy steps use different runAs configuration.
+  # This must be in synchronization with our default configuration.
   echo 'nonroot:x:1000:1000:nonroot:/:/sbin/nologin' > /etc/passwd && \
-  echo 'nonroot:x:1000:' > /etc/group && \
-  mkdir /.docker && chown 1000:1000 /.docker && \
-  mkdir /.ssh && chown 1000:1000 /.ssh
+  echo 'nonroot:x:1000:' > /etc/group
+
+ENV PATH="$PATH:/ko-app"
+ENV HOME="/shared-home"
 
 COPY --from=builder /openshift-builds-git-cloner /ko-app/git
 COPY LICENSE /licenses/
 
-USER 1001
+USER 1000
```

Comment:
Can you please explain why we should use UID 1000 instead of UID 1001? Is there a separate file that references this UID which needs to be kept in sync?

Reply:
The upstream docker file for the "base" mentions this file
https://github.com/shipwright-io/build/blob/df51c7fd49c2fc0494d05e73d0bdea902ab2a4e0/pkg/config/config.go#L91

Reply:
Here is the base..
https://github.com/shipwright-io/build/blob/main/images/base/Dockerfile